client/components/data/query-followers/index.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import isShallowEqual from '@wordpress/is-shallow-equal';
+
+/**
+ * Internal dependencies
+ */
+import { fetchFollowers } from 'calypso/state/followers/thunks';
+
+const REFRESH_INTERVAL = 1000 * 30;
+
+class QueryFollowers extends React.Component {
+	interval = null;
+
+	componentDidMount() {
+		this.props.fetchFollowers( this.props.query );
```

Comment:
This is different to the original flux implementation where we only fetched new data if the store was empty. I think it'd make sense to discuss whether we want to keep it that way.